<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>890</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Exchange">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>791</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:blueviolet;
border:1px solid black;
border-bottom-left-radius:40px;
border-top-right-radius:40px;
color:white;</string>
    </property>
    <property name="title">
     <string>Rate of Exchange</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>691</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid red;
border-radius:0px;
background-color:rgb(105,34,154);
color:balck;</string>
     </property>
     <row>
      <property name="text">
       <string>USDT$</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>EURO</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>IMAM-COIN</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>BTC</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>GOLD</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Low_price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>High_price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Persent_price</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>10</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;</string>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>500</y>
      <width>801</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:yellowgreen;
color:white;
border:1px solid black;
border-bottom-left-radius:40px;
border-top-right-radius:40px;</string>
    </property>
    <property name="title">
     <string>Other</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>631</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
border:1px solid white;
border-radius:0px;
</string>
     </property>
     <row>
      <property name="text">
       <string>Sugar</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Copper</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Cocoa</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Coffee</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Low_price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>High_price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Persent_price</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>10</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;</string>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>953</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
